opendatahub-io/kubeflow
# [WIP] Separate webhook from controller deployment

Resolves #37 

## Description
This PR make the webhook a separate component within the the repo file structure. The built webhook should be buildable into an image and deployed through a separate deployment.

Note: PR in progress, not yet tested.

## How Has This Been Tested?
1. Build the webhook and push to quay.
2. Deploy notebook controller normally.
3. Create a deployment using webhook quay image.
4. Verify no functionality within notebook controller is lost.

## Merge criteria:
